mcastorena0316/finalproject-api
# Back End API for Final Capstone: Tracking APP

Hi there, for this branch I did the back end of the Final Capstone Project, which includes the following 

* REST API that connects to my Front End APP to make a request (POST, GET, DELETE, PATCH)
* Database with 3 tables (user, illness, tracking)
* A session store controller to ain't the session of a user
* Test with RSPEC

Also for the corrections, I implemented the following:
* Add test for failure in every request
* Eliminate username and password for test and development databases
* Create environment keys for password and username for production database
* Clean routes that repeated
* Add test for the disappearance of id when deleting an illness
* Remove test folder

And finally for the second review

* Corrected tests, everyone is working now (attach image)
* Every message is equal to the expected value

![tests](https://user-images.githubusercontent.com/52794673/85627500-390e1400-b634-11ea-8d68-49316fb114d2.png)


Kindly submitting for a review.
Have a good day
